apollo-api/src/main/java/com/apollographql/apollo/api/Operation.java

```diff
@@ -78,6 +78,59 @@
      * @return {@link ResponseFieldMarshaller} to serialize operation data
      */
     ResponseFieldMarshaller marshaller();
+
+    /**
+     * Serializes GraphQL operation response data into its equivalent Json representation.
+     * For example:
+     * <pre>{@code
+     *    {
+     *      "data": {
+     *        "allPlanets": {
+     *          "__typename": "PlanetsConnection",
+     *          "planets": [
+     *            {
+     *              "__typename": "Planet",
+     *              "name": "Tatooine",
+     *              "surfaceWater": 1.0
+     *            }
+     *          ]
+     *        }
+     *      }
+     *    }
+     * }</pre>
+     *
+     * @param indent the indentation string to be repeated for each level of indentation in the encoded document. Must be a string
+     * containing only whitespace,
+     * @param scalarTypeAdapters configured instance of custom GraphQL scalar type adapters
+     * @return json representation of GraphQL operation response
+     */
+    String toJson(@NotNull String indent, @NotNull ScalarTypeAdapters scalarTypeAdapters);
+
+    /**
+     * Serializes GraphQL operation response data into its equivalent Json representation.
+     * For example:
+     * <pre>{@code
+     *    {
+     *      "data": {
+     *        "allPlanets": {
+     *          "__typename": "PlanetsConnection",
+     *          "planets": [
+     *            {
+     *              "__typename": "Planet",
+     *              "name": "Tatooine",
+     *              "surfaceWater": 1.0
+     *            }
+     *          ]
+     *        }
+     *      }
+     *    }
+     * }</pre>
+     *
+     * @param indent the indentation string to be repeated for each level of indentation in the encoded document. Must be a string
+     * containing only whitespace,
+     * @return json representation of GraphQL operation response
+     */
+    String toJson(@NotNull String indent);
```

Comment:
Would it make sense to put this first since the other one overloads this function?